miso-web/src/main/webapp/WEB-INF/pages/editArrayRun.jsp

```diff
@@ -25,6 +25,11 @@ jQuery(document).ready(function() {
 });
 </script>
 
+<c:if test="${arrayRun.id != 0}">
```

Comment:
The Create Array Run page is currently broken. Can change the test to `${pageMode ne 'create'}` to fix (`arrayRun` is not added to the model on the create page)